### PR TITLE
Only copy code if not hidden

### PR DESCRIPTION
**Pull request by orange-elephant**
#690

### CODE REVIEW

File: app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryHolder.java [hunk 1]
```diff
@@ -334,4 +340,12 @@ public void animateCopyText() {
     private void animateAlphaTo(float alpha) {
         itemView.animate().alpha(alpha).setDuration(200).start();
     }
+
+    public boolean isCopied() {
+        return _copied;
+    }
+
+    public void setCopied(boolean copied) {
+        this._copied = copied;
```
Comment (alexbakker) [inline app/src/main/java/com/beemdevelopment/aegis/ui/views/EntryHolder.java:349]:
```java
_copied = copied;
```

**Review by alexbakker — CHANGES_REQUESTED** [review]
Neat! One small nitpick and issue though.

Some entries seem to stay dimmed after highlighting and copying. To reproduce:
- Enable tap to reveal, highlight on tap and copy on tap.
- Have an entry list bigger than the size of the screen, so that you have to scroll down to see the rest.
- Scroll all the way to the top.
- Tap an entry to highlight and reveal it.
- Tap the entry again to copy it.
- Tap the entry again to unhighlight it.
- Scroll all the way to the bottom and notice that some entries are still dimmed.

**Comment by orange-elephant** [comment]
> Some entries seem to stay dimmed after highlighting and copying.

I can't reproduce this on any of my devices, even with 80 entries...
What device is this happening on and how many tokens?

**Comment by alexbakker** [comment]
I reproduced it on a Pixel 5 and this vault file: https://github.com/beemdevelopment/Aegis/blob/599be443697861bc5dc1606fb16c135aac07ef82/app/src/test/resources/com/beemdevelopment/aegis/importers/aegis_plain.json.

Here's a screenrecording of it happening: https://alexbakker.me/u/qegljxgjrs.mp4

**Review by alexbakker — APPROVED** [review]
Looks good, thanks! We'll merge this after the v2.1 release.

**Pull request merged**